hummingbot/strategy/perpetual_market_making/perpetual_market_making.pyx

```diff
@@ -1219,6 +1200,12 @@ cdef class PerpetualMarketMakingStrategy(StrategyBase):
             object order_type = self._close_order_type
 
         if len(proposal.buys) > 0:
+            if position_action == PositionAction.CLOSE:
+                if self._current_timestamp < self._next_buy_exit_order_timestamp:
+                    self.logger().info("Waiting for filled order delay to be over before attempting to close short position(s) again.")
```

Comment:
Will this message logged on every tick while the _next_buy_exit_order_timestamp is not due? If so, I think that's too many of the same message to log.

Reply:
Should I do a timed logger like the spot_perp arb? Or should I remove the log message since we don't display a message for filled order delay anyway?

Reply:
I think you can just remove it, we don't have to log everything all the time.